submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -9,185 +9,144 @@
   </head>
   <body>
     <nav>
-      <div class="nav-container">
-        <div class="nav-el nav-el-text" tabindex="0">
-          <p>Mail</p>
-        </div>
-        <div class="nav-el nav-el-text" tabindex="0">
-          <p>Images</p>
-        </div>
-
-        <!-- #wrap-el - обертка для input + label + popup.
-              CSS к #wrap-el не применял -->
-        <div id="wrap-el">
-
-          <input type="checkbox" id="button-1" tabindex="0">
-          <div class="nav-el" id="nal-el-menu">
+      <ul class="nav-container">
+        <li class="nav-el">
+          <a href="#" tabindex="0">Mail</a>
+        </li>
+        <li class="nav-el">
+          <a href="#" tabindex="0">Images</a>
+        </li>
+
+        <li class="nav-el">
+          <input type="checkbox" id="button-1"  tabindex="0">
+          <a href="#" tabindex="-1">
             <label for="button-1" id="menu-label">
-                <img src="img/apps.jpg" alt="Menu icon" />
+              <img src="img/apps_new.png" alt="Menu icon">
             </label>
-          </div>
-<!-- закончил рефакторинг тут -->
-          <section id="apps-window">
-            <div id="top-triangle"></div>
-            <div id="top-triangle-border"></div>
+          </a>
 
+          <div id="apps-window">
```

Comment:
Avoid styling by `#id` and HTLM tag.
Separate responsibilities:
- HTML tag for markup;
- CSS class for styling;
- ID for JS logic

Reply:
Thank you for the explanation, haven't thought/read about such separate responsibilities before.